index.js

```diff
@@ -226,7 +226,7 @@ app.on('chrome-browser-process-created', function() {
 
         app.commandLine.appendArgument('noerrdialogs');
         process.argv.push('--noerrdialogs');
-        app.quit();
+        app.exit(0);
```

Comment:
Why are we switching from `quit` to `exit`? `quit` seems to be used all over the core. Do all those instances need to be changed as well?

Reply:
Added the following comments in JIRA:

The exception happens because Browser::quit tries to shut services not initialized yet for enable_chromium when 2nd instance of Runtime exits right after start-up.  Browser::exit just exits.

No need to change other calls to quit.